fuzzers/aflplusplus_qemu/builder.Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ARG parent_image
+ARG parent_image=gcr.io/fuzzbench/base-builder
```

Comment:
Can you remove the gcr.io argument in all the files for parent_image, it is not needed after https://github.com/google/fuzzbench/commit/eee4f8d22449d4e6c4345a57c0e81958da4287c4

Reply:
done!